src/server/service/kv.rs

```diff
@@ -57,6 +58,57 @@ pub struct Service<T: RaftStoreRouter + 'static> {
     snap_scheduler: Scheduler<SnapTask>,
     token: Arc<AtomicUsize>, // TODO: remove it.
     recursion_limit: u32,
+    metrics: Metrics,
+}
+
+#[derive(Clone)]
+struct Metrics {
+    kv_get: Histogram,
+    kv_scan: Histogram,
+    kv_prewrite: Histogram,
+    kv_commit: Histogram,
+    kv_cleanup: Histogram,
+    kv_batchget: Histogram,
+    kv_batch_rollback: Histogram,
+    kv_scan_lock: Histogram,
+    kv_resolve_lock: Histogram,
+    kv_gc: Histogram,
+    kv_delete_range: Histogram,
+    raw_get: Histogram,
+    raw_scan: Histogram,
+    raw_put: Histogram,
+    raw_delete: Histogram,
+    coprocessor: Histogram,
+    mvcc_get_by_key: Histogram,
+    mvcc_get_by_start_ts: Histogram,
+    split_region: Histogram,
+}
```

Comment:
Why not use `HashMap<&str, Histogram>`?

Reply:
maybe using fields is faster and more certain than HashMap.

Reply:
and here if using HashMap, we still need to init it one by one, and no need to create some  metrics dynamic, so i use fields